### PR TITLE
Create Aaron U's first post with a trinket and markdown formatting

### PR DESCRIPTION
**Pull request by AU-turtledragon**

### CODE REVIEW

**Comment by AU-turtledragon** [comment]
this reflection is still in progress.

File: _posts/AU-turtledragon/2021-10-20-AU-turtledragon-trinket_post.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+<iframe src="https://trinket.io/embed/python/3c22a3abaa" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+So...
+This code defines a function `avg_PM25()`, which will take the argument `(state)`.
+```
+def avg_PM25(state):
+```
+When called and provided with an argument, the function will call upon the file handle `inFile`, opening the file "uspoll.txt" in `r`ead mode. 
+...to be continued...
```
Comment (hdubbe) [inline _posts/AU-turtledragon/2021-10-20-AU-turtledragon-trinket_post.md]: This is great!

Comment (AU-turtledragon) [reply _posts/AU-turtledragon/2021-10-20-AU-turtledragon-trinket_post.md]: couldn't have done it without you.

**Comment by AU-turtledragon** [comment]
Ok, now this is all done and fully explained.  It's intentionally wordy and thorough just because I felt I needed some practice verbalizing the work of python code.

**Review by MishaCienfuegos — APPROVED** [review]
Great use of markdown! It makes it easier to read the blocks of code and also the inline bits of code like variable names.

**Pull request merged**